src/include/statistics/backend_stats_context.h

```diff
@@ -17,6 +17,7 @@
 #include <sstream>
 #include <unordered_map>
 
+#include "common/statement.h"
```

Comment:
If you just use Statement as a pointer, I think you can fast forward the declaration and include the header in the cpp file. Or it doesn't work with shared_ptr?